src/pages/BeerPage/Components/BeerIngredients/BeerIngredients.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+
+import "./style.sass";
+
+function BeerIngredients({ ingredients }) {
+  return (
+    <div className="ingredients">
+      <h2>Ingredients</h2>
+
+      <h3>Water</h3>
+      <p>25 litres</p>
+
+      <h3>Malt</h3>
+      {ingredients.malt.map((maltItem) => {
```

Comment:
Please, define mapping outside of return block

Reply:
Done